docs/src/development/local/docksal.md

```diff
@@ -1,128 +1,200 @@
+---
+title=Directly in the configuration file
+file=none
+highlight=false
+---
 
-# MySQL Version: 5.6 / 5.7 / 8.0
-#DB_IMAGE='docksal/db:1.2-mysql-5.6'
-DB_IMAGE='docksal/db:1.2-mysql-5.7'
-#DB_IMAGE='docksal/db:1.2-mysql-8.0'
+```txt {location=".docksal/docksal.env"}
+<VARIBLE_NAME>=<VARIABLE_VALUE>
+
+HOSTING_PLATFORM="platformsh"
+```
 
-# PHP Versions Available 5.6 / 7.0 / 7.1 / 7.2
-#CLI_IMAGE='docksal/cli:2.5-php5.6'
-#CLI_IMAGE='docksal/cli:2.5-php7.0'
-CLI_IMAGE='docksal/cli:2.5-php7.1'
-#CLI_IMAGE='docksal/cli:2.5-php7.2'
+{{< /codetabs >}}
+
+To apply your changes, run the following command:
+
+```bash
+fin project restart
 ```
 
-You can further create and customize a `.docksal/docksal.yml` file within your project.
-This is a docker-compose file and can be customized as needed for your application,
-as some customizations are specific to certain applications.
-See Docksal documentation on [extending stock images](https://docs.docksal.io/stack/extend-images).
+### Customize your stack
 
-## Downloading MySQL data from Platform.sh into Docksal
+Docksal comes configured with a default PHP container (known as the CLI container),
+web container, and database container.
+You can customize the versions for each.
 
-In most cases, downloading data from Platform.sh and loading it into your project is straightforward.
-The following commands, run from your application root, download a compressed database backup
-and load it into the local Docksal database container.
+Check the available versions for each by running `fin image registry <IMAGE>`.
+For example, to check the available containers for the app, run `fin image registry docksal/cli`.
+
+You can then use the version you want by adding it as an [environment variable](#add-environment-variables).
+For example, to use the latest version, run the following command:
 
 ```bash
-fin platform db:dump --gzip -f /tmp/database.sql.gz
-fin exec 'zcat < /tmp/database.sql.gz | mysql -u user -puser -h db default'
+fin config set CLI_IMAGE='docksal/cli:latest'
 ```
 
-### Connecting Projects to the Database
+You can also customize the `.docksal/docksal.yml` file.
+Use it for a docker-compose definition for your [custom configurations](https://docs.docksal.io/stack/custom-configuration/#custom-configuration).
 
-After importing your database into the project the next step is connecting to the database server.
-The following information can be used for setting up a connection for your application.
+### Import MySQL data from Platform.sh into Docksal
 
-Key | Value
-----|-----
-DB Name | default
-Username | user
-Password | user
-Host | db
-Port | 3306
+To download your data from Platform.sh and load it into your Docksal database container, run the following commands:
 
-See the [exporting tutorial](/tutorials/exporting.md) for information on how to use rsync.
+```bash
+fin platform db:dump --gzip -f /tmp/database.sql.gz
+fin exec 'zcat < /tmp/database.sql.gz | mysql -u user -puser -h db default'
```

Comment:
❓Might be worth to add variables there
```suggestion
fin exec 'zcat < /tmp/database.sql.gz | mysql -u <USERNAME> -p <PASSWORD> -h db <DATABASE_NAME>'
```
and point to [where the values can be retrieved](https://docs.docksal.io/service/db/access/).

Reply:
The link is to where they can be set rather than retrieved, but could help. It might be a little more advanced than I'd want to fit in a tutorial, especially a side note at the bottom. How about a link to where they could be set as a compromise?

Reply:
😍  I like the additional context you added, it should make it clearer how to adapt and use the command when needed.